external-crates/move/crates/move-vm-types/src/values/values_impl.rs

```diff
@@ -4345,11 +4358,16 @@ impl Container {
                     return None;
                 }
                 match &v[0] {
-                    ValueImpl::Address(a) => A::MoveValue::Signer(*a),
+                    ValueImpl::Address(a) => A::MoveValue::Signer(**a),
                     _ => return None,
                 }
             }
             (_layout, _val) => return None,
         })
     }
 }
+
+#[test]
+fn assert_sizes() {
+    assert_eq!(size_of::<ValueImpl>(), 16);
+}
```

Comment:
I wanted to put this in value_tests, but it would require changing the visibility of `ValueImpl`